controllers/reconcilers/kserve_networkpolicy_reconciler.go

```diff
@@ -37,108 +34,24 @@ type KserveNetworkPolicyReconciler struct {
 	client               client.Client
 	scheme               *runtime.Scheme
 	networkPolicyHandler resources.NetworkPolicyHandler
-	deltaProcessor       processors.DeltaProcessor
 }
 
 func NewKServeNetworkPolicyReconciler(client client.Client, scheme *runtime.Scheme) *KserveNetworkPolicyReconciler {
 	return &KserveNetworkPolicyReconciler{
 		client:               client,
 		scheme:               scheme,
 		networkPolicyHandler: resources.NewNetworkPolicyHandler(client),
-		deltaProcessor:       processors.NewDeltaProcessor(),
 	}
 }
 
 func (r *KserveNetworkPolicyReconciler) Reconcile(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
-
-	// Create Desired resource
-	desiredResource, err := r.createDesiredResource(isvc)
-	if err != nil {
-		return err
-	}
-
-	// Get Existing resource
-	existingResource, err := r.getExistingResource(ctx, log, isvc)
-	if err != nil {
-		return err
-	}
-
-	// Process Delta
-	if err = r.processDelta(ctx, log, desiredResource, existingResource); err != nil {
-		return err
-	}
-	return nil
-}
-
-func (r *KserveNetworkPolicyReconciler) createDesiredResource(isvc *kservev1beta1.InferenceService) (*v1.NetworkPolicy, error) {
-	desiredNetworkPolicy := &v1.NetworkPolicy{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      networkPolicyName,
-			Namespace: isvc.Namespace,
-			Labels: map[string]string{
-				"app.kubernetes.io/version":               "release-v1.9",
-				"networking.knative.dev/ingress-provider": "istio",
-			},
-		},
-		Spec: v1.NetworkPolicySpec{
-			PodSelector: metav1.LabelSelector{},
-			Ingress: []v1.NetworkPolicyIngressRule{
-				{
-					From: []v1.NetworkPolicyPeer{
-						{
-							NamespaceSelector: &metav1.LabelSelector{
-								MatchLabels: map[string]string{
-									"name": "openshift-user-workload-monitoring",
-								},
-							},
-						},
-					},
-				},
-			},
-			PolicyTypes: []v1.PolicyType{v1.PolicyTypeIngress},
-		},
-	}
-	return desiredNetworkPolicy, nil
+	return r.DeleteNetworkPolicy(ctx, isvc.Namespace)
 }
 
 func (r *KserveNetworkPolicyReconciler) getExistingResource(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) (*v1.NetworkPolicy, error) {
```

Comment:
function `getExistingResource` is not called from any place. We could remove this function as well.